lib/modules/datasource/maven/util.ts

```diff
@@ -485,6 +485,20 @@ export async function getDependencyInfo(
     }
   }
 
+  const relocation = pomContent.descendantWithPath(
+    'distributionManagement.relocation',
+  );
+  if (relocation) {
+    const relocationGroup =
+      relocation.valueWithPath('groupId') ?? dependency.group;
+    const relocationName =
+      relocation.valueWithPath('artifactId') ?? dependency.name;
+    result.replacementName = relocationGroup + ':' + relocationName;
+    const relocationVersion = relocation.valueWithPath('version');
+    result.replacementVersion = relocationVersion ?? version;
```

Comment:
It would be wrong, if the version will not be set.
The [documentation](https://maven.apache.org/ref/3.9.9/maven-model/maven.html#class_relocation) says the following:
> If any of the values are omitted, it is assumed to be the same as it was before.

Reply:
i see `replacementVersion ` is currently required by reovate 😕